src/Features/CSharp/Portable/SignatureHelp/TupleConstructionSignatureHelpProvider.cs

```diff
@@ -181,13 +181,11 @@ private IEnumerable<SignatureHelpParameter> ConvertTupleMembers(INamedTypeSymbol
             var result = new List<SignatureHelpParameter>();
             foreach (var element in tupleType.TupleElements)
             {
-                var type = element.GetTypeWithAnnotatedNullability();
-
                 // The display name for each element. 
                 // Empty strings for elements not explicitly declared
                 var elementName = element.IsImplicitlyDeclared ? string.Empty : element.Name;
 
-                var typeParts = type.ToMinimalDisplayParts(semanticModel, position).ToList();
+                var typeParts = element.Type.ToMinimalDisplayParts(semanticModel, position).ToList();
```

Comment:
❔ Do we need to do anything here to get the nullability information to show with the display parts?

Reply:
The code change here doesn't change requirements, and it appears this is already working per the existing tests.